src/cmd/service.go

```diff
@@ -85,6 +85,11 @@ var listServiceCmd = &cobra.Command{
 		cobra.CheckErr(err)
 		if isJsonOutput() {
 			common.JsonPrint(json.MarshalIndent(list, "", "    "))
+		} else if isCsvOutput() {
+			fmt.Printf("%s\n", strings.Join([]string{"NAME", "ID", "ALIASES"}, ","))
+			for _, item := range list {
+				fmt.Printf("%s,%s,[%s]\n", item.Name, item.Id, strings.Join(item.Aliases, ","))
```

Comment:
Add `[]` around aliases since it was hard to differentiate it from NAME and ID fields but might not be necessary.

Reply:
hmm - is that valid CSV?  I think the extra comma's from `strings.Join(item.Aliases, ",")` will cause problems

Reply:
Yeah, I think you're right.  Sounds like we would need to use a different delimiter there.

Reply:
>Remember that the enclosing character (typically double quotes) must be used when required, such as when the delimiter appears in a field.

Reply:
ya to be safe i'd use `|` or `/` or any other character you can think of.  We just need to standardize it across the board.

Reply:
Ok, updated it to use the CSV writer but that defaults to quotes if there's a comma in a field.  Looking at the output, using `/` makes it look a bit like a file path but the `|` blends in a bit more so not sure which would be better.  Went with `/` for now.